readme.md

```diff
@@ -49,7 +49,8 @@ store.uploadExisting(myZipFile, token).then(res => {
 ```javascript
 const target = 'default'; // optional. Can also be 'trustedTesters'
 const token = 'xxxx'; // optional. One will be fetched if not provided
-store.publish(target, token).then(res => {
+const deployPercentage = 25; // optional. Will default to 100%.
+store.publish({ target, deployPercentage, token }).then((res) => {
```

Comment:
On second thought, I'm not ready to release a breaking version. I already have a partial rewrite in progress and I'd want to release _that_ instead. See #84 

Can you revert this API to accept a third parameter?

Reply:
So it should be change this way if I understand correctly:
```
store.publish(target, token, deployPercentage)
```

Reply:
Yes

Reply:
Here it is: https://github.com/fregante/chrome-webstore-upload/pull/93